### ISSUE TITLE
PAA

### ISSUE THREAD

**Issue by TonyBagnall** [alan-turing-institute/sktime#93]
hi, 

I'll post thoughts on each algorithm as I prepare it for assimilation. Will post them here for discussion with Matthew, and will do some black box testing

1. Do we need to fit at all? I see no reason, since the transform is data driven
2. What happens when the intervals are not integer length? Seems to just make the last interval shorter?
3. This may suggest a more python efficient way of doing it.
        http://vigne.sh/posts/piecewise-aggregate-approx/

**Comment by MatthewMiddlehurst** [comment]
Would be good to get everything fixed up for integration as you said, we can have a discussion next week regarding it if that’s good with you. For the PAA stuff:
1. No, it was added mostly to conform with the other transformers I did, but as the BaseTransformer has a default which basically does the same thing it can be removed.
2. If the instance doesn’t divide cleanly the frames should still be equal in size, the remaining variable is tracked for this and splits a value between 2 frames in this situation.
3. Will have a look and compare, will need a pythonise most of it likely (good learning experiance at least).
